### PR TITLE
#238: calculate total number of unique active contributors

### LINKED ISSUE #238: `total_active_contributors` into DoT
After #203 

Let's add `total_active_contributors` metric into `dimensions-of-terrain.rb`, which should mean the total number of unique contributors to all repositories, who made at least one commit during the last 30 days.

### PR DESCRIPTION
**Pull request by Yegorov**
Closes #238

### CODE REVIEW

**Comment by Yegorov** [comment]
@Suban05 please review

File: judges/dimensions-of-terrain/dimensions-of-terrain.rb [hunk 1]
```diff
@@ -103,3 +103,14 @@
...
+# Total number of unique active contributors to all repos
+active_contributors = Set.new
+Fbe.unmask_repos.each do |repo|
+  Fbe.octo.search_commits(
+    "repo:#{repo} author-date:>#{(f.when - (30 * 24 * 60 * 60)).iso8601[0..9]}"
+  )[:items].each do |commit|
+    active_contributors << commit.dig(:author, :id)
+  end
+end
+f.total_active_contributors = active_contributors.count { |c| !c.nil? }
```
Comment (Suban05) [inline judges/dimensions-of-terrain/dimensions-of-terrain.rb:116]: @Yegorov maybe it's better use `compact`: `active_contributors.compact.count`
or check for `nil`, when we add values to `active_contributors`?

**Review by Suban05 — COMMENTED** [review]
@Yegorov looks good. added a small comment

File: judges/dimensions-of-terrain/dimensions-of-terrain.rb (hunk 1 above)
Comment (Yegorov) [reply judges/dimensions-of-terrain/dimensions-of-terrain.rb]: @Suban05 Ok, probably it is better to check at once for null. Thanks, fixed!

**Review by Suban05 — APPROVED** [review]
👍

**Comment by Yegorov** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@Yegorov thanks!

**Comment by 0crat** [comment]
@Suban05 Great job on the review! 🌟 You've earned +4 points: +4 base, -10 for limited comments, but +10 as a minimum reward. Your updated balance is now [+138](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the good work and aim for more detailed feedback next time!

**Comment by 0crat** [comment]
@Yegorov Hey there, awesome work on your contribution! 🚀 You've scored +10 points for this one: +4 for showing up, +9 for your 196 [hits-of-code](https://www.yegor256.com/2014/11/14/hits-of-code.html) (just shy of that 200 mark!), but we had to deduct 4 because there were only 4 review comments. Remember, more review comments can boost your score next time! [Keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) the code coming - you're doing great! Your current balance is [+176](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Let's aim for even more points on your next contribution!

**Pull request merged**